upgrade/capsule.py

```diff
@@ -88,15 +145,17 @@ def satellite6_capsule_setup(sat_host, os_version, upgradable_capsule=True):
     copy_ssh_key(sat_host, cap_hosts)
     # Dont run capsule upgrade requirements for n-1 capsule
     if upgradable_capsule:
+        save_server_config(sat_host)
         execute(sync_capsule_repos_to_upgrade, cap_hosts, host=sat_host)
         for cap_host in cap_hosts:
+            os.environ["ROBOTTELO_UPGRADE__CAPSULE_HOSTNAME"] = cap_host
             execute(add_baseOS_repo, baseurl, host=cap_host)
         for cap_host in cap_hosts:
             logger.info('Capsule {} is ready for Upgrade'.format(cap_host))
     return cap_hosts
 
 
-def satellite6_capsule_upgrade(cap_host, sat_host):
+def satellite_capsule_upgrade(cap_host, sat_host):
```

Comment:
```suggestion
def capsule_upgrade(cap_host, sat_host):
```
Confuses with both names.